.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "docs/sphinx-resources"]
```

Comment:
question: To avoid confusion on the future should the submodule also be renamed?